src/page/driver/login/components/PageInput.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState } from "react";
+import style from "./PageInput.module.css";
+
+type InputType = {
+  value: string;
+  inputHeandler: (e: React.ChangeEvent<HTMLInputElement>) => void;
+  validation: boolean;
+};
+
+export const EmailInputComponent = ({
+  value,
+  inputHeandler,
+  validation,
+}: InputType) => {
+  return (
+    <div className={style.container}>
+      <span className={style.text}>이메일</span>
+      <input
+        className={`${style.input} ${
+          style[value && validation ? "complete" : ""]
+        } ${style[value && !validation ? "invalid" : ""]} `}
+        value={value ?? ""}
+        onChange={inputHeandler}
+        type="text"
+        id="email"
+        name="email"
+      />
+      {validation ? null : (
+        <span className={style.invaild}>이메일 형식이 아닙니다.</span>
+      )}
+    </div>
+  );
+};
+
+export const PasswordInputComponent = ({
+  value,
+  inputHeandler,
+  validation,
+}: InputType) => {
+  const [invisible, setInvisible] = useState<Boolean>(true);
```

Comment:
```suggestion
  const [invisible, setInvisible] = useState<boolean>(true);
```
Boolean은 `JavaScript` 의 객체 타입으로, `TypeScript` 에서 사용할 때 권장되지 않습니다..!
따라서 `TypeScript` 에서의 primitive 타입인 boolean을 사용하는 게 어떨까요?

Reply:
prettier에서 경고해서 변경했습니다.